fastlane/CommonFastfile

```diff
@@ -0,0 +1,270 @@
+before_all do
+  update_fastlane
+end
+
+desc "Bump version, edit changelog, and create pull request"
+lane :bump do |options|
+  ensure_git_branch(branch: options[:branch] || 'main')
+  ensure_git_status_clean
+
+  # Ensure GitHub API token is set
+  if ENV['GITHUB_PULL_REQUEST_API_TOKEN'].nil?
+    UI.error("Environment variable GITHUB_PULL_REQUEST_API_TOKEN is required to create a pull request")
+    UI.error("Please make a fastlane/.env file from the fastlane/.env.SAMPLE template")
+    UI.user_error!("Could not find value for GITHUB_PULL_REQUEST_API_TOKEN")
+  end
+
+  # Get and print current version number
+  version_number = current_version_number
+  UI.important("Current version is #{version_number}")
+
+  # Ask for new version number
+  new_version_number = UI.input("New version number: ")
+
+  generated_contents = github_changelog(options)
+  changelog_path = edit_changelog(generated_contents: generated_contents)
+  changelog = File.read(changelog_path)
+  
+  create_new_release_branch(version: new_version_number)
+  replace_version_number(version: new_version_number)
+
+  attach_changelog_to_master(new_version_number)
+
+  commit_updated_files_and_push(version: new_version_number)    
+  
+  create_pull_request(
+    title: "Release/#{new_version_number}",
+    base: "main",
```

Comment:
This doesn't really work for hotfixes. But I think this is ok for now, we can add more automation in the future. This is how it worked until now in the iOS repo